packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerItemInput.tsx

```diff
@@ -0,0 +1,5 @@
+import { TextInput } from '@/ui/input/components/TextInput';
+
+export const NavigationDrawerItemInput = () => {
```

Comment:
Not used?

Reply:
it is being used! it is the input that appears on creating a new folder.

Reply:
nope! not used!